bouncer/package.json

```diff
@@ -46,5 +46,8 @@
     "tsx": "^4.7.2",
     "typescript": "^5.4.5"
   },
+  "engines": {
+    "pnpm": ">=9"
```

Comment:
throw an error if the installed pnpm version is too low

Reply:
My pnpm and node js were out of date. This may of been my issue.